appeals/api/src/server/endpoints/documents/documents.controller.js

```diff
@@ -316,17 +316,15 @@ const updateDocumentsAvCheckStatus = async (req, res) => {
 	const { body } = req;
 	try {
 		const documents = body.documents;
+		await documentRepository.createDocumentAvStatus(documents);
+
 		for (const document of documents) {
 			const latestDocument = await documentRepository.getDocumentWithAllVersionsById(document.id);
```

Comment:
Do we need to fetch all versions here?

Reply:
I think we do, as the scan will not necessarily update the latest version

Reply:
Can we not just fetch the version we want to update? Or even just call update and ignore not found errors?